ArcGISRuntimeSDKQt_CppSamples/Routing/NavigateRoute/NavigateRoute.cpp

```diff
@@ -120,55 +120,52 @@ void NavigateRoute::setMapView(MapQuickView* mapView)
 
 void NavigateRoute::connectRouteTaskSignals()
 {
-  connect(m_routeTask, &RouteTask::solveRouteCompleted, this, [this](const QUuid&, const RouteResult& routeResult)
+  connect(m_routeTask, &RouteTask::doneLoading, this, [this](const Error& error)
   {
-    if (routeResult.isEmpty())
-      return;
-
-    if (routeResult.routes().empty())
-      return;
-
-    m_routeResult = routeResult;
-    m_route = qAsConst(m_routeResult).routes()[0];
+    if (error.isEmpty())
```

Comment:
This should be changed to reduce the indenting level.

Rather than
```c++
if (error.isEmpty())
{
  // do a lot of stuff
}
```

Try this:
```c++
if (!error.isEmpty())
  return;

// do a lot of stuff that's not already indented one level
```

Reply:
Modified wherever applicable.